gammapy/maps/axes.py

```diff
@@ -3007,8 +3047,8 @@ def pix_to_coord(self, pix):
         idx = np.round(pix).astype(int)
         return self._labels[idx]
 
-    def coord_to_idx(self, coord, **kwargs):
-        """Transform labels to indices
+    def coord_to_idx(self, coord):
+        """Transform labels to indices.
```

Comment:
label to index or coordinate to index?

This need to be clarified throughout the docstring

Reply:
Well the thing is that `LabelMapAxis` basically is a container for string representation (using our MapAxis framework let say). But then, coordinates don't really exist... At least that's how I understand it. 
But we could just be more general and just say `axis coordinates` as here :
`    def pix_to_coord(self, pix):
        """Transform pixel to axis coordinates.`
What do you think about this ?

Reply:
I think it's more than the naming of the function is odd, given what it does. So maybe only changing it slightly, how about:
`Transform coordinate labels to indices`?

Reply:
Seems good to me !